src/v2/client/completeLoginFlow.js

```diff
@@ -0,0 +1,50 @@
+/*!
+ * Copyright (c) 2020, Okta, Inc. and/or its affiliates. All rights reserved.
+ * The Okta software accompanied by this notice is provided pursuant to the Apache License, Version 2.0 (the "License.")
+ *
+ * You may obtain a copy of the License at http://www.apache.org/licenses/LICENSE-2.0.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *
+ * See the License for the specific language governing permissions and limitations under the License.
+ */
+
+import Enums from 'util/Enums';
+import Errors from 'util/Errors';
+import { toQueryString } from '@okta/okta-auth-js';
+
+import { clearTransactionMeta } from './transactionMeta';
+
+export async function completeLoginFlow (settings, idxResponse) {
```

Comment:
Should we add default values here to be safe?

Reply:
in this case both parameters are required. It could throw. But this method is not exposed on the OktaSignIn object and is not expected to be called directly. (it was refactored out of BaseLoginRouter)